app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -119,7 +119,7 @@ impl Background {
         let inset = selection_size + selection_offset;
         let shape = Rectangle();
         shape.set_corner_radius(RADIUS);
-        shape.set_border(selection_size);
+        shape.set_border(-selection_size);
```

Comment:
It's not obvious at the call site what a negative border means. Would this API be clearer if the use of signs to implement the different behaviors were an implementation detail internal to `Rectangle`?

Reply:
Changed the public API, now the negatives are completely hidden away. Instead, we have three different methods to use three kinds of borders.